.ci/README.md

```diff
@@ -0,0 +1,67 @@
+# data.table continuous integration and deployment
+
+On each Pull Request opened in GitHub we run Travis CI and Appveyor to provide prompt feedback about the status of PR. Our main CI pipeline runs on GitLab CI. GitLab repository automatically mirrors our GitHub repository and runs pipeline on `master` branch. It tests more environments and different configurations. It publish variety of artifacts.
+
+## Environments
+
+### [GitLab CI](./../.gitlab-ci.yml)
+
+Test jobs:
+- `test-rel-lin` - `r-release` on Linux, most comprehensive test environment, `-O3 -flto`, extra check for no compilation warnings, includes testing [_with other packages_](./../inst/tests/other.Rraw) ([extended suggests](./../inst/tests/tests-DESCRIPTION))
+- `test-rel-cran-lin` - `--as-cran` on Linux, `-g0`, extra check for `Status: OK` `R CMD check` result
+- `test-dev-cran-lin` - `r-devel` and `--as-cran` on Linux, `--enable-strict-barrier --disable-long-double`
+- `test-rel-vanilla-lin` - `r-release` on Linux, no suggested deps, no OpenMP, `-O0`, tracks memory usage during tests
+- `test-310-cran-lin` - R 3.1.0 on Linux
+- `test-rel-win` - `r-release` on Windows
+- `test-dev-win` - `r-devel` on Windows
+- `test-rel-osx` - MacOSX build not yet deployed, see [#3326](https://github.com/Rdatatable/data.table/issues/3326) for status
+
+Artifacts:
+- [homepage](https://rdatatable.gitlab.io/data.table) - made with [pkgdown](https://github.com/r-lib/pkgdown)
+- [html manual](https://rdatatable.gitlab.io/data.table/library/data.table/html/00Index.html)
+- [pdf manual](https://rdatatable.gitlab.io/data.table/web/packages/data.table/data.table.pdf)
+- [html vignettes](https://rdatatable.gitlab.io/data.table/library/data.table/doc/index.html)
+- R packages repository for `data.table` and all _Suggests_ dependencies, url: `https://Rdatatable.gitlab.io/data.table`
+  - sources
+  - Windows binaries for `r-release` and `r-devel`
+- [CRAN-like homepage](https://rdatatable.gitlab.io/data.table/web/packages/data.table/index.html)
+- [CRAN-like checks results](https://rdatatable.gitlab.io/data.table/web/checks/check_results_data.table.html) - note that all artifacts, including this page, are being published only when all test jobs successfully pass, thus one will not see an _ERROR_ status there (unless `allow_failure` option has been used in a job).
```

Comment:
is this new? or did i miss something. this is awesome!

Reply:
Nope, it is there for a long time already. It was useful during memtest, there was a link to a png chart of memory usage during tests. It has been removed since then, now only csv is linked.